src/force/nep3.cu

```diff
@@ -271,7 +273,11 @@ static __global__ void find_descriptor(
         }
         accumulate_s(d12, x12, y12, z12, gn12, s);
       }
-      find_q_nep3(paramb.n_max_angular + 1, n, s, q + (paramb.n_max_radial + 1));
+      if (paramb.num_L > paramb.L_max) {
+        find_q_with_4body(paramb.n_max_angular + 1, n, s, q + (paramb.n_max_radial + 1));
+      } else {
+        find_q(paramb.n_max_angular + 1, n, s, q + (paramb.n_max_radial + 1));
```

Comment:
If I understand correctly, this function computes the descriptor q for a single atom, using pre-computed values for the basis functions( YLM & C4B)?

Reply:
Yes! YLM and C3B are \{C^{3b}_k\}_{k=0}^{23} and \{C^{4b}_k\}_{k=0}^{4} in the manuscript I sent you.

Reply:
Actually it would be better to change the name YLM to C3B.